### DISCUSSION TITLE
What is wrong with this script?

### DISCUSSION THREAD

**Discussion by 14Codes** [community/community#75683] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

var name = prompt("What is your name?");
console.log("Hello, " + name + "!");

var num1 = prompt("Enter the first number:");
var num2 = prompt("Enter the second number:");
var sum = num1 + num2;
console.log("The sum of the numbers is: " + sum);

I ran it on a testing software and some errors showed up.

**Comment by OfficialChrisTucker** [comment, marked as answer]
I have done some pr request on some of  your projects before. Here is a script I think should work. 

var name = prompt("What is your name?");
console.log("Hello, " + name + "!");

var num1 = parseFloat(prompt("Enter the first number:"));
var num2 = parseFloat(prompt("Enter the second number:"));

if (!isNaN(num1) && !isNaN(num2)) {
  var sum = num1 + num2;
  console.log("The sum of the numbers is: " + sum);
} else {
  console.log("Invalid input. Please enter numeric values.");
}

**Reply by 14Codes** [reply]
OH I SEE WHAT I DID. I DIDN'T DO A SUM OF NUMBERS LOL THANKS

**Comment by 14Codes** [comment]
Uhm, So.. I need help again @OfficialChrisTucker 


`function calculateSum(a, b) {
  return a + b;
}

var result = calculateSum(5, 10);
console.log("The sum is: " + result);

function calculateAverage(numbers) {
  var sum = 0;
  for (var i = 0; i < numbers.length; i++) {
    sum += numbers[i];
  }
  return sum / numbers.length;
}

var numbers = [2, 4, 6, 8, 10];
var average = calculateAverage(numbers);
console.log("The average is: " + average);

function findLargestNumber(numbers) {
  var max = 0;
  for (var i = 0; i <= numbers.length; i++) {
    if (numbers[i] > max) {
      max = numbers[i];
    }
  }
  return max;
}

var numbers = [1, 3, 5, 7, 9];
var largestNumber = findLargestNumber(numbers);
console.log("The largest number is: " + largestNumber);
`

**Reply by QuocHuy03** [reply]
function calculateSum(a, b) {
  return a + b;
}

var result = calculateSum(5, 10);
console.log("The sum is: " + result);

function calculateAverage(numbers) {
  var sum = 0;
  for (var i = 0; i < numbers.length; i++) {
    sum += numbers[i];
  }
  return sum / numbers.length;
}

var numbersForAverage = [2, 4, 6, 8, 10];
var average = calculateAverage(numbersForAverage);
console.log("The average is: " + average);

function findLargestNumber(numbers) {
  var max = 0;
  for (var i = 0; i < numbers.length; i++) {
    if (numbers[i] > max) {
      max = numbers[i];
    }
  }
  return max;
}

var numbersForLargest = [1, 3, 5, 7, 9];
var largestNumber = findLargestNumber(numbersForLargest);
console.log("The largest number is: " + largestNumber);

**Reply by OfficialChrisTucker** [reply]
hm

**Comment by 14Codes** [comment]
the script doesn't work. Let me open another ticket sorry for the issue @OfficialChrisTucker

**Comment by OfficialChrisTucker** [comment]
10-4

**Comment by OfficialChrisTucker** [comment]
just add me so we can discuss there.
